spec/system/placements/schools/placements/view_placements_list_spec.rb

```diff
@@ -141,4 +172,8 @@ def expect_to_see_table_headings
       expect(page).to have_content "Start date"
     end
   end
+
+  def and_i_click_on(link_text)
+    click_on link_text
+  end
```

Comment:
Sorry to be nitpicky but for proper "Given, When, Then" style syntax, this should be "when I click on"

```suggestion
  def when_i_click_on(link_text)
    click_on link_text
  end
```